pandas/tests/frame/test_analytics.py

```diff
@@ -784,17 +799,13 @@ def alt(x):
         assert kurt.name is None
         assert kurt2.name == 'bar'
 
-    def _check_stat_op(self, name, alternative, frame=None, has_skipna=True,
+    def _check_stat_op(self, name, alternative, main_frame, float_frame,
```

Comment:
Is there any risk here in using arguments identical to the fixture names? Assuming since this method itself is not a test that pytest won't try to inject the fixture here but not sure if that will always be the case

Reply:
Changed. Did you seem my comment about breaking up this function?